### PR TITLE
Add unit test script for CI

### PR DESCRIPTION
**Pull request by nateglims**
*Description of changes:* Add a shell script to run unit tests in CI. Unit tests in this case are binaries generated by CMake with the `-DBUILD_TESTS=1` option set and have `utest` in their name. This distinction is to differentiate from platform integration tests, service integration tests and demos.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: ci/ci_utest.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+    if [ $? -ne "0" ] ; then
+        STATUS=1
+    fi
+
+done
+
+if [ "$STATUS" = "1" ] ; then
+    exit 1
+fi
```
Comment (yourslab) [inline ci/ci_utest.sh:25]: Clean 👌🏼👌🏼👌🏼

File: ci/ci_utest.sh [hunk 2]
```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```
Comment (sarenameas) [inline ci/ci_utest.sh:1]: I am cool with the ci folder, but maybe this should be under tools/ci?

Comment (nateglims) [reply ci/ci_utest.sh]: Done

**Pull request merged**